cli/aeon/cmd/transport.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"fmt"
+
+	"golang.org/x/exp/maps"
+)
+
+// AeonTransport is an auxiliary type to constrain the list of stored value.
```

Comment:
Is a type implemented something cobra-related? Please, add a note about it if so.